src/main/java/net/masterthought/cucumber/reducers/ReportFeatureByIdMerger.java

```diff
@@ -0,0 +1,30 @@
+package net.masterthought.cucumber.reducers;
+
+import net.masterthought.cucumber.json.Feature;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+final class ReportFeatureByIdMerger implements ReportFeatureMerger {
+
+    @Override
+    public List<Feature> merge(List<Feature> features) {
+        Map<String, Feature> mergedFeatures = new HashMap<>();
+        for (Feature feature : features) {
+            Feature mergedFeature = mergedFeatures.get(feature.getId());
+            if (mergedFeature == null) {
+                mergedFeatures.put(feature.getId(), feature);
+            } else {
+                mergedFeatures.get(feature.getId()).addElements(feature.getElements());
```

Comment:
would it merge or add?
based on the implementation addElements I guess it adds so after that some elements might be duplicated
https://github.com/damianszczepanik/cucumber-reporting/pull/880/files#diff-8bf6b44089f1dce48374088686c23f67R50